Arrangement_on_surface_2/include/CGAL/Arr_tags.h

```diff
@@ -90,7 +90,7 @@ struct Validate_left_side_category< GeometryTraits_2, false > {
   void missing__Left_side_category()
   { 
     T
-      missing__Left_side_category__assuming__Arr_oblivious_side_tag__instead; 
+      missing__Left_side_category__assuming__Arr_oblivious_side_tag__instead __attribute__((unused)); 
```

Comment:
In `<CGAL/config.h>`, we have a macro `CGAL_UNUSED` that can be used. Its value is `__attribute__ ((__unused__))` for gcc and clang, and empty for other compilers.